WalletWasabi.Fluent/ViewModels/Wallets/Receive/ReceiveAddressViewModel.cs

```diff
@@ -36,7 +36,7 @@ public ReceiveAddressViewModel(Wallet wallet, HdPubKey model)
 
 		GenerateQrCode();
 
-		SetupCancel(enableCancel: false, enableCancelOnEscape: true, enableCancelOnPressed: true);
+		SetupCancel(enableCancel: false, enableCancelOnEscape: true, enableCancelOnPressed: true, escapeGoesBack: true);
```

Comment:
nACK.

I am sure on this screen ESC should close the dialog. The user just generated/opened an address to use it, why would they want to go back?

Reply:
Reverted the change for ReceiveAddressViewModel